Engine/Graphics/Outdoor.cpp

```diff
@@ -3172,7 +3182,7 @@ void ODM_ProcessPartyActions() {
             // pParty->pPartyBuffs[PARTY_BUFF_WATER_WALK].uOverlayID + 119] |=
             // 1u;
             v79 =
-                (int)&stru_5E4C90_MapPersistVars._decor_events
+                (short *)&stru_5E4C90_MapPersistVars._decor_events
```

Comment:
indexing as a short* will require dividing whats inside []  by two, isnt it? or indexing operator takes precedence?

Reply:
Indexing takes precedence afaik, but I will confirm. Also I could find the same variable with same type cast and same indexing in another function ODM_ProcessPartyActions, which makes me believe it's correct.

Reply:
Confirmed indexing takes precedence.